### ISSUE TITLE
Pressing Tab to create second heading not working on either edit mode in Obsidian 0.13.14 Mac

### ISSUE THREAD

**Issue by bjorndotwork** [tgrosinger/advanced-tables-obsidian#142]
**Describe the bug**
When creating a new table according to Advanced Tables docs' instructions, Tab key doesn't create second heading as expected.

**To Reproduce**
1. Create new note.
2. Type `| Heading 1` and press Tab.

**Expected behavior**
I expect Advanced Tables to respond by appending "| |" and putting cursor between the two pipes.

**Desktop (please complete the following information):**
 - OS: macOS 12.1
 - Obsidian Version: 0.13.14
 - Advanced Tables Version: 0.15.0

**Comment by Caddickbrown** [comment]
I believe this is related to the Live Preview View

**Comment by 1activegeek** [comment]
I would concur, I believe it has to do with the new Live Preview update in the editor. I'm seeing the same behavior.

**Comment by staticaland** [comment]
Yes. Settings → Options → Editor → Use legacy editor as a temporary workaround.

**Comment by WAcry** [comment]
It is about legacy editor option, which only works with source mode, rather than the new LIve Mode

**Comment by scpedicini** [comment]
Can confirm that the new "Live Preview" feature seems to break Advanced Table functionality.

**Comment by Elaws** [comment]
Similar to this [issue](https://github.com/tgrosinger/advanced-tables-obsidian/issues/147). @michaelyang02 found a workaround : just add something below the table, even a blank line.

**Comment by AlexGoico** [comment]
Same on Windows for Obsidian 0.14.2. Using the legacy editor that @staticaland said also makes it work on Windows so the plugin is broken for "Live Preview" in general.

**Comment by mochsner** [comment]
Looks like "Next cell" can't even be bound to tab on Ubuntu for Obsidian v.0.15.9
